helm-charts/mach-operator/values.yaml

```diff
@@ -0,0 +1,239 @@
+
+    operator_server_ip_port_addr: localhost:8091
+
+    # the layer1 chain id the avs contracts in
+    layer1_chain_id: 
+
+    # the layer2 chain id
+    layer2_chain_id:
+  genesis.json: |
+    {}
+  mach-config.toml: |
+    ##############################################
+    # Mach alerter configuration                 #
+    ##############################################
+    [alerter]
+    # Only print the warning log if the alerter is disabled,
+    # Otherwise the alerts will be sent to the Mach AVS Operator
+    # Optional, default value: false
+    enable = true
+    # The JSONRPC endpoint of Mach AVS Operator
+    url = "http://127.0.0.1:8901"
+
+    ##############################################
+    # Mach validator configuration               #
+    ##############################################
+    [validator]
+    # Validate mode: "unsafe", "safe", "strict"
+    #
+    # "unsafe": Don't execute EVM, return ok directly.
+    # "safe": Execute EVM ，return ok regardless of whether it's right or wrong.
+    # "strict": Execute EVM, return the execution result.
+    #
+    # Optional, default value: "unsafe"
+    mode = "unsafe"
+    # The rate limit of rpc request
+    # Optional, default value: 100ms
+    rpc_rate_ms = 100
+
+    ##############################################
+    # Mach fetcher configuration                 #
+    ##############################################
+    [fetcher]
+    # The JSONRPC endpoint of Optimism node
+    url = ""
+    # The interval (milliseconds) for fetching finalized/latest block of Optimism node
+    # Optional, default value: 1000ms
+    interval_ms = 4000
+
+    ##############################################
+    # Mach scheduler configuration               #
+    ##############################################
+    [scheduler]
+    # The timeout waiting for verify result.
+    # If it times out, the verify result will be checked again in the next tick.
+    # Optional, default value: 100ms
+    verify_timeout_ms = 100
+
+    ##############################################
+    # Mach prometheus configuration              #
+    ##############################################
+    [prometheus]
+    # The TCP port of Prometheus exporter
+    # Optional, default value: 9094
+    port = 9094
+    # The HTTP URI path for fetching Prometheus metrics
+    # Optional, default value: "/metrics"
+    metrics_path = "/metrics"
+    # The indication if expose Prometheus exporter on all interfaces
+    # true: listen address = "0.0.0.0:port"
+    # false: listen address = "127.0.0.1:port"
+    # Optional, default value: false
+    external = true
+node:
+  image:
+    repository: public.ecr.aws/altlayer/mach-operator
+    pullPolicy: IfNotPresent
+    tag: "latest"
+  resources:
+    limits:
+      cpu: 4
+      memory: 4Gi
+    requests:
+      cpu: 2
+      memory: 4Gi
+  args:
+    - --config=/app/config/operator.yaml
+  env:
+    - name: OPERATOR_BLS_KEY_PASSWORD
```

Comment:
That's cool we can use values for setting the password, but a much better way is to allow also use `volume` and `volumeMounts` too, because, for example, I want to use secrets for passing the bls keys and passwords here like that:

```yaml
node:
  env:
    - name: OPERATOR_BLS_KEY_PASSWORD
      valueFrom:
        secretKeyRef:
          name: avs-bls-key
          key: bls-private-key-password
  volumeMounts:
    - name: bls-key
      mountPath: /app/operator_keys/bls_key.json
      subPath: bls_key.json
      readOnly: true


volumes:
  - name: bls-key
    secret:
      secretName: avs-bls-key
      items:
        - key: bls-private-key
          path: bls_key.json

```
Can you add it too, please?